src/infra/watcher/watcher.go

```diff
@@ -0,0 +1,163 @@
+package watcher
+
+import (
+	"context"
+	"log/slog"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/contre95/soulsolid/src/features/importing"
+	"github.com/fsnotify/fsnotify"
+)
+const DEBOUNCE_SECS = 5
```

Comment:
You can unexport this as it's only used in the pkg to camel case. 

```suggestion
const debounceSeconds = 5
```